### ISSUE TITLE
Waydroid (Failed to get service waydroidplatform, trying again...) / Orange Pi 5

### ISSUE THREAD

**Issue by pr4x77** [waydroid/waydroid#776]
Hi, 

when I wanna start the full ui using "waydroid show-full-ui" I repeatedly getting the error "Failed to get service waydroidplatform, trying again...".

Im using Waydroid GAPPS 20230218 on Armbian with GNOME Wayland. Hardware is a Orange Pi 5 with linux-5.10.110-rockchip-rk3588 (nightly build from 23.02.23) as kernel.

System Datetime = 1676652158
Vendor Datetime = 1676674354

Waydroid Container and Session are running. Output of echo $XDG_SESSION_TYPE is "wayland"

I already tried to completely reinstall and to rebuild follwing this  [instruction](https://github.com/waydroid/waydroid/issues/427#issuecomment-1188806996)

[waydroid_dmesg.log](https://github.com/waydroid/waydroid/files/10813883/waydroid_dmesg.log)
[waydroid.log](https://github.com/waydroid/waydroid/files/10813884/waydroid.log)
[waydroid_base.prop.txt](https://github.com/waydroid/waydroid/files/10813888/waydroid_base.prop.txt)
[waydroid.prop.txt](https://github.com/waydroid/waydroid/files/10813892/waydroid.prop.txt)

Anyone can help with this?

Thanks!

**Comment by aleasto** [comment]
Your GPU is unsupported in mesa, you will need to force software rendering: https://docs.waydro.id/faq/get-waydroid-to-work-through-a-vm

**Comment by pr4x77** [comment]
Doesnt it work with panfork mesa? Is there any chance that its getting supported?

**Comment by aleasto** [comment]
No. Support needs to land in upstream mesa.

**Comment by develperbayman** [comment]
yeah this issue basically leaves any graphical apps a no go software rendering sucks ....you have to use swiftshader in waydroid.prop change from mesa to swiftshader .......why not openGL?
